runtime/obj.c

```diff
@@ -118,7 +118,6 @@ CAMLprim value caml_obj_with_tag(value new_tag_v, value arg)
   } else {
     res = caml_alloc_shr(sz, tg);
     for (i = 0; i < sz; i++) caml_initialize(&Field(res, i), Field(arg, i));
-    res = caml_check_urgent_gc(res);
```

Comment:
Why removing this call?

Reply:
BTW, this one is relatively important since the calls to `caml_initialize` can create a lot of back-references, which can require a minor collection.

And since this is a ocaml-only function which allocate, we can make it polling. Even in the case where it allocates in minor heap.

Reply:
I thought about this, and just assumed that the original code which did not include it was correct. I did not get into details but this seemed plausible to me given that we are making a copy of something that is already there. Let me know what you think, because in to know independently of polling callback.

Reply:
Ok, that's what I wanted to hear. So it was good to ask.

I did not ask for a specific PR or a changes entry. When I try to understand the code, I am a big fan of `C-x v g` (`vc-annotate`), so either a comment in the code or a line in the commit log can help in this situation. (I say that in case people believe that commit logs are no longer useful in the github days.)